go.mod

```diff
@@ -2,20 +2,6 @@ module github.com/docker/compose-cli
 
 go 1.15
 
-// the distribution version from ecs plugin is quite old and it breaks containerd
```

Comment:
You got me, I thought all were removed, but you just moved them below! :D

Reply:
Yeah, I tried to make it cleaner :)

Reply:
I bumped docker to v20.10.0-rc2 so this should now be a bit cleaner.